lib/Doctrine/DBAL/Platforms/SQLServerPlatform.php

```diff
@@ -815,7 +815,21 @@ public function getListTablesSQL()
     {
         // "sysdiagrams" table must be ignored as it's internal SQL Server table for Database Diagrams
         // Category 2 must be ignored as it is "MS SQL Server 'pseudo-system' object[s]" for replication
-        return "SELECT name FROM sysobjects WHERE type = 'U' AND name != 'sysdiagrams' AND category != 2 ORDER BY name";
+        // Table names are returned with the schema name IF and only if the schema
+        // is not the default schema for the user accessing the database.
+        return "SELECT CASE 
+        WHEN s.NAME <> p.default_schema_name
+            THEN '[' + s.NAME + '].[' + t.NAME + ']'
```

Comment:
@zeroedin-bill I don't think this is a good idea to always quote here. Those identifiers should go through `quoteIncomingIdentifier()` aswell IMO as you might otherwise get problems with casing and such.
Maybe it would be a good idea to retrieve the schema name and table name seperately here and treat them accordingly in the schema manager. Thoughts?